### PR TITLE
vncviewer: add page

### PR DESCRIPTION
**Pull request by LachlanMarnham**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

issue: #2393

### CODE REVIEW

File: pages/linux/vncviewer.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+- Launch a VNC client which connects to a host on a given display:
+
+`vncviewer {{host}}:{{display_number}}`
+
+- Launch in full-screen mode:
+
+`vncviewer -FullScreen {{host}}:{{display_number}}`
+
+- Launch a VNC client with specific screen geometry:
```
Comment (sbrl) [inline pages/linux/vncviewer.md]: Perhaps .....`with a specific`..... would read better?

File: pages/linux/vncviewer.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# vncviewer
+
+> Launches a VNC (Virtual Network Computing) client.
+
+- Launch a VNC client which connects to a host on a given display:
+
+`vncviewer {{host}}:{{display_number}}`
```
Comment (agnivade) [inline pages/linux/vncviewer.md]: From where does one get the display_number ?

File: pages/linux/vncviewer.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+`vncviewer -FullScreen {{host}}:{{display_number}}`
+
+- Launch a VNC client with specific screen geometry:
+
+`vncviewer --geometry {{width}}x{{height}} {{host}}:{{display_number}}`
+
+- Launch a VNC client which connects to a host on a given port:
+
+`vncviewer {{host}}::{{port}}`
```
[the remark this answers is not in the document]
Comment (LachlanMarnham) [reply pages/linux/vncviewer.md]: @agnivade Intentional. If one types host:number, then number is parsed as a display number. If one types host::number, number is parsed as a port.

File: pages/linux/vncviewer.md (hunk 1 above)
Comment (LachlanMarnham) [reply pages/linux/vncviewer.md]: @sbrl yes I think so, I will make the change.

File: pages/linux/vncviewer.md (hunk 2 above)
Comment (LachlanMarnham) [reply pages/linux/vncviewer.md]: @agnivade It is the display that the VNC server is running on. I erred away from explaining this concept due to the line in the contributing guidelines "...and avoid explaining general UNIX concepts...". I also decided to keep this consistent with vncserver.md, which had been submitted by someone else already, and which refers to the display_number without definition. ie:

- Kill an instance of VNC Server running on a specific display:
`vncserver --kill :{{display_number}}`

Perhaps I should reword to "Launch a VNC client which connects to a host which is serving on a given display", to make it clear? Any ideas?

Comment (agnivade) [reply pages/linux/vncviewer.md]: Yes, I saw the vncserver.md page. Now that I think a bit more, I think this should be fine. I initially thought this to be some vnc specific concept. But if it is a general UNIX concept, we can let it be.

Thanks for the clarification.

**Pull request merged**